src/torchjd/autojac/_transform/stack.py

```diff
@@ -10,15 +10,12 @@
 
 class Stack(Transform[_A, Jacobians]):
```

Comment:
Should it be `Transform[Gradients, Jacobians]` ? We cannot stack anything else than gradients right ?

Reply:
I dont think so.

The return type of the contained transforms should always be `Gradients`, but their input type can be anything, as long as its the same for all of them.

Reply:
Which is why the type of `transforms` is `transforms: Sequence[Transform[_A, Gradients]]`

Reply:
Yeah of course, my bad